dev.py

```diff
@@ -305,6 +305,14 @@ def lint(filenames, php, js):
         service_trigger('nodejs', 'npm run lint:js')
 
 
+@cli.command()
+def run_tests():
+    if _service_is_running('composer'):
+        service_trigger('composer', './vendor/bin/phpunit')
+    else:
+        info('Need composer service to be running. Have you run the up command?')
```

Comment:
The composer service needs to be running in order to run tests. Have you run the up command?